test/resolver.test.ts

```diff
@@ -1,23 +1,24 @@
 import { resolveAppFunction } from "../src/helpers/resolve-app-function";
+import { describe, expect, vi, it } from "vitest";
 
-const stubAppFnPath = require.resolve("./fixtures/plugin/stub-plugin");
+const stubAppFnPath = require.resolve("./fixtures/plugin/stub-plugin.ts");
```

Comment:
Should this `require` be removed?
`require` isn't defined in an ESM environment, and all other instances have been removed

Reply:
WIth what should it be replaced?

Reply:
> Should this `require` be removed? `require` isn't defined in an ESM environment, and all other instances have been removed

We're not ESM-Only yet, but we can remove it in the future. We can use [`createRequire`](https://nodejs.org/api/module.html#modulecreaterequirefilename) as a workaround once we're trule ESM. The ESM-native `import.meta.resolve` was only stabilized in Node v20.6.0.